### DISCUSSION TITLE
Add Editor toggle for snaking sliders

### DISCUSSION THREAD

**Discussion by kongehund** [ppy/osu#31061] [Ideas]
A user may have different preferences for slider snaking depending on whether he's playing or mapping.

While playing, snaking out sliders might be preferable i.e. to reduce visual clutter to make next objects more visible.
While editing, snaking out sliders makes the shape of the previous slider invisible, making it less convenient to match visuals of new objects to previous objects.

Similar preference differences may exist for snaking in sliders.

I suggest adding toggles to the `View` menu

Stable:
![image](https://github.com/user-attachments/assets/b0133b03-0829-4ba1-89f2-da0e288bba6d)
Lazer (Snaking out disabled in settings):
![image](https://github.com/user-attachments/assets/39e116bb-907d-435c-8c32-23485fa6fea9)
Lazer (Snaking out enabled in settings):
![image](https://github.com/user-attachments/assets/038206aa-a861-409b-afc7-069fb92f362f)

**Comment by peppy** [comment]
Once we have settings profiles, I think there can be an "editor profile" which allows adjusting ALL game wide settings for when the user is in the editor. This will cover things like skin choice as well.
